content/en/blog/grpc-csharp-future.md

```diff
@@ -1,5 +1,4 @@
 ---
-draft: true
 title: The future of gRPC in C# belongs to grpc-dotnet
 date: 2021-05-01
```

Comment:
Will you be publishing this today? If so, we need to change the date to today (since the website doesn't currently publish future-dated blog entires).

Reply:
No I was planning to publish on Monday.

Reply:
FYI, I've added a note to that effect in the opening comment.